ee/modules/160-multitenancy-manager/openapi/values.yaml

```diff
@@ -9,11 +9,11 @@ properties:
       projects:
         items:
           properties:
-            projectTypeName:
+            templateName:
               type: string
             projectName:
               type: string
-            params:
+            templateValues:
```

Comment:
Hmm, names seem invalid after renaming other fields.

Reply:
Corrected and added a description block for ProjectTemplate